SocketRocket/ProxyConnect.h

```diff
@@ -0,0 +1,6 @@
+#import <Foundation/Foundation.h>
+
+@interface ProxyConnect : NSObject
+-(instancetype)initWithURL:(NSURL *)url;
```

Comment:
Surround the declaration of `init` and `openNetworkStream` with 2 empty lines, per common style.